checks/raw/fuzzing.go

```diff
@@ -105,14 +110,55 @@ var languageFuzzSpecs = map[clients.LanguageName]languageFuzzConfig{
 				"and test that specific properties are satisfied."),
 	},
 	clients.TypeScript: {
-		filePattern: "*.ts",
+		filePatterns: []string{"*.ts"},
```

Comment:
does it make sense to merge JS and TS now that multiple patterns are supported? or keep them separate?

Reply:
I think C and C++ have a similar question. Some fuzzers will support both, i assume some fuzzers might be specific to one vs the other?